rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
```

Comment:
Don't add a `rust-toolchain` file when targeting stable because it prevents people from building the project with Rust nightly. If you need to override your current settings, use `rustup override`.

Reply:
Precisely the idea of this file is that there is a standard version to compile the project so that there are no users who have features that can break the project or break the original flow